wiki/release/release-process.md

```diff
@@ -592,10 +549,13 @@ svn ls https://dist.apache.org/repos/dist/release/pulsar
 svn rm https://dist.apache.org/repos/dist/release/pulsar/pulsar-2.Y.0
 ```
 
-## 21. Move branch to next version
+## Move release branch to next version
 
-Follow the instructions in step 6, but run the script on the release branch.
+Run the following commands in release branches.
 
 ```shell
 ./src/set-project-version.sh 2.X.Y-SNAPSHOT
+
+git commit -m 'Bumped version to 2.X.Y-SNAPSHOT' -a
```

Comment:
```suggestion

git add .
git commit -s -m 'Bumped version to 2.X.Y-SNAPSHOT'
```

In case of the files aren't tracked yet.

Reply:
No, we should not. Because `set-project-version.sh` will generate a new temporary file `deployment/terraform-ansible/deploy-pulsar.yaml-e` and unfortunately it's not included in `.gitignore`.

Reply:
I think we should add them to `.gitignor` instead, lol. But fair enough to keep as is.

Reply:
Yes, we should.